app/models/observation.rb

```diff
@@ -407,27 +407,54 @@ class Observation < AbstractModel
           )
           return none unless box.valid?
 
-          # expand box by epsilon to create leeway for Float rounding
+          # resize box by epsilon to create leeway for Float rounding
           # Fixes a bug where Califoria fixture was not in a box
           # defined by the fixture's north, south, east, west
-          expanded_box = box.expand(0.00001)
+          resized_box = box.expand(0.00001)
 
           if box.straddles_180_deg?
             where(
-              (Observation[:lat] >= expanded_box.south).
-              and(Observation[:lat] <= expanded_box.north).
-              and(Observation[:long] >= expanded_box.west).
-              or(Observation[:long] <= expanded_box.east)
+              (Observation[:lat] >= resized_box.south).
+              and(Observation[:lat] <= resized_box.north).
+              and(Observation[:long] >= resized_box.west).
+              or(Observation[:long] <= resized_box.east)
             )
           else
             where(
-              (Observation[:lat] >= expanded_box.south).
-              and(Observation[:lat] <= expanded_box.north).
-              and(Observation[:long] >= expanded_box.west).
-              and(Observation[:long] <= expanded_box.east)
+              (Observation[:lat] >= resized_box.south).
+              and(Observation[:lat] <= resized_box.north).
+              and(Observation[:long] >= resized_box.west).
+              and(Observation[:long] <= resized_box.east)
             )
           end
         }
+  scope :outside_box, # Use named parameters (n, s, e, w), any order
+        lambda { |**args|
+          box = Mappable::Box.new(
+            north: args[:n], south: args[:s], east: args[:e], west: args[:w]
+          )
+          return none unless box.valid?
+
+          # resize box by epsilon to create leeway for Float rounding
+          resized_box = box.expand(-0.00001)
+
+          if box.straddles_180_deg?
```

Comment:
Do the current where conditions below actually differ?  Specifically is there any difference between `or.where` and `or`?  Also I think the first case is wrong.  Shouldn't it be long < west AND long > east (since in this case west > east)?

Reply:
Tried creating a project called "Pacific Ocean" that includes the entire Pacific Ocean as an example and the ProjectsController#show method fails on line 444.  I think this is a blocker and this needs a test.

Reply:
@mo-nathan Thanks for looking at this again.
I'll look at this and the bug you found (projects that span the international date line currently fail.)